scripts/skills/perks/perk_rf_unstoppable.nut

```diff
@@ -131,7 +125,7 @@ this.perk_rf_unstoppable <- ::inherit("scripts/skills/skill", {
 				id = 15,
 				type = "text",
 				icon = "ui/icons/warning.png",
-				text = ::Reforged.Mod.Tooltips.parseString(::MSU.Text.colorRed("Will cause you to lose all stacks of [Unstoppable|Skill+perk_rf_unstoppable]"))
+				text = ::Reforged.Mod.Tooltips.parseString(::MSU.Text.colorRed(format("Will cause you to lose all stacks of [%s|Perk+%s]", ::Const.Perks.findById(this.getID()).Name, split(::IO.scriptFilenameByHash(this.ClassNameHash), "/").top())))
```

Comment:
I've seen this way of writing a few times now and it strikes me as lazy and hard to read.

I recommend that you create a global utility function like this:
```
function getPerkHyperlink( _class )
{
   return format([%s|Perk+%s]", ::Const.Perks.findById(_class.getID()).Name, split(::IO.scriptFilenameByHash(_class.ClassNameHash), "/").top())
}

Reply:
I agree it would be good to have a function like that. I'll set it up.